app/views/core_induction_programmes/modules/show.html.erb

```diff
@@ -32,6 +32,21 @@
             <% end %>
           </h2>
 
+          <% if lesson.teacher_standards_for(current_user) %>
+            <details class="govuk-details  govuk-!-margin-top-2 govuk-!-margin-bottom-2" data-module="govuk-details">
+              <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                View which teachers' standards this topic relates to
+              </span>
+              </summary>
```

Comment:
whitespace in details class 

Indenting on span? or does it sit like that? Feel like i've seen this before with span but is there something different about it's set like this?